schemas/doi.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "DOI",
+  "description": "Persistent identifier",
+  "required": [
+    "contextId",
+    "doi"
+  ],
+  "properties": {
+    "contextId": {
+      "type": "integer",
+      "description": "Context id for DOI",
+      "apiSummary": true
+    },
+    "doi": {
+      "type": "string",
+      "description": "Persistent identifier",
+      "apiSummary": true,
+			"validation": [
+				"regex:/^\\d+(.\\d+)+\\//"
+			]
+    },
+    "id": {
+      "type": "integer",
+      "apiSummary": true,
+      "readOnly": true
+    },
+		"resolvingUrl": {
+			"type": "string",
+			"description": "DOI URL that resolves to the indicated item",
+			"apiSummary": true,
+			"readOnly": true
+		},
+    "status": {
+      "type": "integer",
+      "description": "Persistent identifier status",
+      "apiSummary": true,
+      "default": 1,
+      "validation": [
+        "in:1,2,3,4,5"
+      ]
+    }
+  }
+}
```

Comment:
To add confusion to things, the `.json` files use tabs instead of spaces. You may need to tell your editor to respect the configuration in each project: https://github.com/pkp/ojs/blob/main/.editorconfig

Reply:
These files should use tabs instead of spaces, so there shouldn't be a need for `indent_size`. I think if we added `indent_size` of 4, your editor might just us four spaces instead of a tab? It looks like maybe Sublime Text requires a plugin to use `.editorconfig`... https://github.com/sindresorhus/editorconfig-sublime